CMakeLists.txt

```diff
@@ -29,11 +29,17 @@ set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS_RELEASE} -flto")
 set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO "${CMAKE_EXE_LINKER_FLAGS_RELEASE} -flto")
 
 if(APPLE)
+if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "arm64") # Apple Silicon Processor
+set(BACKEND_TARGET_TRIPLE "aarch64-apple-darwin")
+set(BREW_HOME "/opt/homebrew")
+else()	
 set(BACKEND_TARGET_TRIPLE "x86_64-apple-darwin")
+set(BREW_HOME "/usr/local")
+endif()
 if(NOT USE_NIX)
-include_directories(AFTER SYSTEM /usr/local/include)
-link_directories(AFTER /usr/local/lib)
-set(ENV{PKG_CONFIG_PATH} "/usr/local/opt/libffi/lib/pkgconfig")
+include_directories(AFTER SYSTEM /opt/homebrew/include)
```

Comment:
Looks like you forgot to use BREW_HOME here

Reply:
Thanks for spotting this, it is fixed now.